selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -1394,8 +1394,26 @@ public void waitSuccessCondition(ExpectedCondition<Boolean> expression) {
     waitSuccessCondition(expression, DEFAULT_TIMEOUT);
   }
 
-  public void closeCurrentWinAndReturnToMainTab(String mainTabHandle) {
+  /**
+   * wait url with specified text fragment
+   *
+   * @param url fragment
+   */
+  public void waitUrlContains(String url) {
+    webDriverWaitFactory.get().until(ExpectedConditions.urlContains(url));
+  }
+
+  /**
+   * wait specified url in the browser
+   *
+   * @param url expected url
+   */
+  public void waitUrlToBe(String url) {
+    webDriverWaitFactory.get().until(ExpectedConditions.urlToBe(url));
+  }
+
+  public void closeCurrentWinAndReturnToPreviewsOne(String mainWindowHandle) {
```

Comment:
`mainWindowHandle` is confusing a bit so as it's ordinary window handle. 
So, next signature will be more consistent, IMHO `closeCurrentWindowAndSwitchToAnother(String windowToSwitch)`